### DISCUSSION TITLE
Can't Get Google OAuth to Work

### DISCUSSION THREAD

**Discussion by komposer-aml** [supabase/supabase#9632] [Questions] [answered]
Hello,

I would like to offer my users the ability to login through their Google accounts.

However, I keep getting error 500s in the console each time I try to login. I attached the code invoking the sign-in, the console output, and auth logs. As far as I know, I have added the domain to my Google dashboard and setup the callback URL from Supabase, but I will redo those steps again to make sure.

Any other ideas on how to get this to work? I don't understand what could possibly be the issue at this point.

![Screen Shot 2022-10-17 at 4 44 02 AM](https://user-images.githubusercontent.com/41843630/196301680-95893168-5ff4-4622-93f0-a3babc798659.png)
<img width="461" alt="Screen Shot 2022-10-17 at 4 42 03 AM" src="https://user-images.githubusercontent.com/41843630/196301682-19e0741b-5326-4873-9e30-a66eef5f1585.png">
![Screen Shot 2022-10-17 at 4 41 12 AM](https://user-images.githubusercontent.com/41843630/196301684-114054fd-4c15-4f39-ac28-4e103f7eaad3.png)
<img width="546" alt="Screen Shot 2022-10-17 at 7 26 08 PM" src="https://user-images.githubusercontent.com/41843630/196301936-098972f7-4332-41f2-a20c-a74a21da8d11.png">

**Comment by GaryAustin1** [comment, marked as answer]
Normally the first thing to look at with new user 500 errors is any trigger function on auth.users insert.  
But... there is currently this that has shown up a few times in Discord in the past week.  [Get user API endpoint returns HTTP 500 with supabase-js v2.0.0](https://github.com/supabase/gotrue-js/issues/491)

**Comment by komposer-aml** [comment]
Hello Gary, my man! I believe you answered my Discord discussion post when I was trying to make my trigger functions on auth.users insert

I couldn't find anything concerning in my Postgres logs, but just in case, here is my new auth.user record DB function:

function handle_new_user:
begin
  insert into "userProfiles" (id, email)
  values (new.id, new.email);
  return new;
end;

I am concerned it might be related to the bug you linked as their outputs seem to be the exact same as mine.

**Reply by GaryAustin1** [reply]
If you did not see anything in Postgres logs, then that points to that new issue as I "THINK" it is an error in gotrue before it even gets to Postgres.  You could certainly turn off your trigger to verify, but usually a trigger error shows up in the logs and at quick glance your code looks ok.
